advertise.md

```diff
@@ -0,0 +1,86 @@
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+{% include JB/setup %}
+<div class="advertise">
+  <div class="advertise container">
+    <h2>Technologies</h2>
+    <div class="border row">
+      <div class="border col-md-4 col-sm-4" style="height:200px;">
+        <div class="panel-content">
+          <label class="_default-text" style="margin-top: 20px;">
+            <img src="./assets/themes/zeppelin/img/spark_logo.jpg" width="140px">
+          </label>
+          <label class="_hover-text">
+            Spark interpreter group which consists of five interpreters which are Spark, Pyspark, SparkR, Spark SQL, Dependency loader.<br/>
+            <a href="./docs/0.8.0-SNAPSHOT/quickstart/spark_with_zeppelin.html" class="panel-button">USE NOW <span class="glyphicon glyphicon-chevron-right"></span></a>
+          </label>                     
+        </div>
+      </div>
+      <div class="border col-md-4 col-sm-4" style="height:200px;">
+        <div class="panel-content">
+          <label class="_default-text" style="font-size:56px;">SQL</label>
+          <label class="_hover-text">
+            JDBC interpreter lets you create a JDBC connection to any data sources seamlessly. Postgresql, Mysql, MariaDB, Redshift, Apache Hive and so on.<br/>
+            <a href="./docs/0.8.0-SNAPSHOT/quickstart/sql_with_zeppelin.html" class="panel-button">USE NOW <span class="glyphicon glyphicon-chevron-right"></span></a>
+          </label>
+        </div>
+      </div>
+      <div class="border col-md-4 col-sm-4" style="height:200px;">
+        <div class="panel-content">
+          <label class="_default-text" style="margin-top:65px;">
```

Comment:
5. need to increase `margin-top` for the python logo. It's not aligned with the other image (spark) and text (SQL)

### Current

![image](https://cloud.githubusercontent.com/assets/4968473/26777034/aa6941fc-4a16-11e7-8c3f-e88638d2d2ce.png)




### Fixed: `margin-top: 75px`

![image](https://cloud.githubusercontent.com/assets/4968473/26777023/9d70f422-4a16-11e7-9cc8-49038131da7d.png)